scripts/init/database/migrations/001.do.sql

```diff
@@ -13,7 +13,7 @@ CREATE TABLE organizations (
 
 /* TODO: need policies unique constraint on org_id, name */
 CREATE TABLE policies (
-  id          SERIAL UNIQUE,
+  id          VARCHAR(128) UNIQUE,
```

Comment:
Is a new migration file actually needed ?
Given that this is being used and potentially has been deployed in production, changing an existing migration file will make postgrator refuse to update the schema with these new changes.